tests/integration/test_iam_idempotent.py

```diff
@@ -1,4 +1,5 @@
 from datetime import date
+import time
```

Comment:
Is the `time` import still needed here?  I don't see any specific reference to it within this file.

Reply:
:doh: I'd added a time.sleep when I was trying to debug this. Good catch

Reply:
fixed